src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/SwitchHelper.groovy

```diff
@@ -138,28 +131,28 @@ class SwitchHelper {
             }
             northbound.getLinks(sw.dpId, null, null, null).each {
                 if (sw.features.contains(SwitchFeature.NOVIFLOW_COPY_FIELD)) {
-                    multiTableRules.add(encodeIslVxlanEgress(it.source.portNo))
-                    multiTableRules.add(encodeIslVxlanTransit(it.source.portNo))
+                    new PortColourCookie(CookieType.MULTI_TABLE_ISL_VXLAN_EGRESS_RULES, it.source.portNo).getValue()
+                    new PortColourCookie(CookieType.MULTI_TABLE_ISL_VXLAN_TRANSIT_RULES, it.source.portNo).getValue()
```

Comment:
`multiTableRules.add` ?

Reply:
oops, I've already fixed the same issue in another line)
thx, fixed